tests/pytorch/iris.py

```diff
@@ -9,8 +9,6 @@
 import torch.nn.functional as F
 from packaging.version import Version
 
-SUMMARY_WRITER = torch.utils.tensorboard.SummaryWriter()
```

Comment:
`torch.utils.tensorboard.SummaryWriter` generates logs in a `runs` directory in the current working directory.